### ISSUE TITLE
Campaign integration not passing checkbox fields

### ISSUE THREAD

**Issue by AndrewMac** [verbb/formie#191]
**Description**

To begin with, I'm not sure if this is an issue with Formie or Campaign! But ...
I have a few checkbox fields in Formie that are mapped to Campaign user fields in the Integration config. Unfortunately, though, the values in those fields don't get sent through to Campaign. I've tried matching the value in Formie field values to both the field names and values in Campaign, checking capitalisation, but neither works. Plain-text field types do come through — just not checkbox fields (and, it seems, radio buttons).

If you think this might be an issue with Campaign rather than Formie, please let me know and I'll post over there.

**Steps to reproduce**
1. Set up checkbox fields in both Formie and Campaign
2. Set up integration with Campaign
3. Try to send those fields to Campaign. Nothing comes through (at least, for me!)

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.2.24
- Campaign plugin version: 1.16.4
- Craft version: Craft Pro 3.5.14
- Multi-site: Yes

**Additional context**

**Comment by engram-design** [comment]
I'd say this if a Formie issue, so will double check! I didn't exhaustively test every field type, but I probably should've.

**Comment by engram-design** [comment]
Alrighty, so I'm just testing this and I can't seem to reproduce it. For radio and checkbox fields, you'll need to ensure the values of your options match exactly.

For example, the field options for the Craft field:
<img width="984" alt="Screen Shot 2020-10-27 at 9 49 22 pm" src="https://user-images.githubusercontent.com/1221575/97291645-4d8e4180-189e-11eb-9a44-9d0ada380581.png">

And the field options for the Formie field:
<img width="821" alt="Screen Shot 2020-10-27 at 9 50 01 pm" src="https://user-images.githubusercontent.com/1221575/97291706-61d23e80-189e-11eb-8534-82d13b40e40e.png">

(you can label options to be whatever you like).

The form submission:
<img width="423" alt="Screen Shot 2020-10-27 at 9 50 41 pm" src="https://user-images.githubusercontent.com/1221575/97291786-7c0c1c80-189e-11eb-8326-3f9aefb2651b.png">

The resulting contact:
<img width="680" alt="Screen Shot 2020-10-27 at 9 51 15 pm" src="https://user-images.githubusercontent.com/1221575/97291851-8fb78300-189e-11eb-9d7b-7e30013ec364.png">

**Comment by AndrewMac** [comment]
Thanks again for looking into this. I've double-checked that the field values match, which they do. But I notice that your Formie submission screenshot didn't include a **checkbox** field, which is the one that's not working for me. The radio buttons are, as you note, working as expected (I think I might have accidentally misled you on that).

I can also confirm that a Formie radio button submits successfully to both checkbox and radio button field types in Campaign (if that makes sense).

**Comment by engram-design** [comment]
Oh, sorry I thought you meant both Radio and Checkboxes weren't working - they're very similar. Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.24",
  "...": "..."
}
```

Then run `composer update`.

Also - be sure to re-save your form's integration settings, just to be safe.

**Comment by AndrewMac** [comment]
My fault entirely, so need to apologise. I did at first think it was both, and I misled you completely on that. So I am the one who needs to apologise and say that I'm sorry this took more of your time than it should have.

Thanks so much for sorting it out, though 👍

**Comment by engram-design** [comment]
Fixed in [1.2.25](https://github.com/verbb/formie/releases/tag/1.2.25)
